tests/bluechi_test/test.py

```diff
@@ -166,6 +175,7 @@ def run(self, exec: Callable[[BluechiControllerContainer, Dict[str, BluechiNodeC
             exec(ctrl_container, node_container)
         finally:
             try:
+                self.gather_coverage(ctrl_container, node_container)
```

Comment:
I think we should gather coverage only when we build RPMs with coverage support. So I'd probably add following:

1. Define tmt environment variable `WITH_COVERAGE` with default value `no` similarly as in https://github.com/containers/bluechi/pull/488/files#diff-ec5e35f9a00ac177713459ba4f15ab7a84faa6d69adb1d05a2134f2f15ea0dc5R10
2. Create `with_coverage` fixture which will expose `WITH_COVERAGE` value to the test code similary as in https://github.com/containers/bluechi/blob/main/tests/bluechi_test/fixtures.py#L50
3. Run coverage gathering only when `with_coverage` returns `True`